### ISSUE TITLE
One service start/stop other services

### ISSUE THREAD

**Issue by ldeveloperl1985** [GluuFederation/gluu-gateway#385]
One service starting or stopping other services. Facing this issue in CentOS and REHL7.

Expected Behaviour:
`konga` service should only start konga UI. `kong` service should only stop/start kong.

Actual Behaviour:
`konga` service starting UI as well as starting `kong` also. `kong` service starting/stopping kong as well as starting/stopping OXD/DB/KongaUI

**Comment by nikdavnik** [comment]
kong is requrement for konga service, seems konga can not start is kong is not running.
thats why when you stop one another is also being stopped.

**Comment by ldeveloperl1985** [comment]
I think, remove this dependency for every individual service, In case if a user wants to start/stop every services, they have a `gluu-gateway` service.

**Comment by nikdavnik** [comment]
@ldeveloperl1985 can konga run when kong and oxd-server are down?

**Comment by ldeveloperl1985** [comment]
@nikdavnik yes konga runs, it will throw error message to check oxd server so it is ok to remove services dependency. For whole services start/stop we have `gluu-gateway` service.
